packages/react/src/SubNav/SubNav.tsx

```diff
@@ -158,19 +158,24 @@ const _SubNavRoot = memo(({id, children, className, 'data-testid': testId, fullW
 type SubNavHeadingProps = {
   href: string
   'data-testid'?: string
+  as?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6'
```

Comment:
You could Pick the tags from Heading, as they're already exported there?

https://github.com/primer/brand/blob/main/packages/react/src/Heading/Heading.tsx#L8

Reply:
Good suggestion, thanks